RELEASING.md

```diff
@@ -0,0 +1,39 @@
+# Release process
+
+## Steps
+
+1. Checkout the repository on the correct branch and changeset (`master`)
+2. Run pre-release script
+
+    1. Release candidate
+
+       In case of release candidate, run localy commmand `make VERSION=x.v.z RELEASE_CANDIDATE=1 pre-release`
+
+       For example `make VERSION=0.3.0 RELEASE_CANDIDATE=2 pre-release` will generate the release version `0.3.0-rc.2`
+
+    2. Final release
+
+       Run localy the command `make VERSION=x.v.z pre-release`
+
+3. Commit all the changes generated from the previous command:
+
+   ```console
+   $ git add .
+   $ git commit -s -m "release vX.Y.X"
+   # or
+   $ git commit -s -m "release vX.Y.X-rc.W"
+   ```
+
+4. Uncomment the job `push_latest_to_docker_hub` in `.gitlab-ci.yml` in case you plan to push the image to dockerhub with the `latest` image tag.
```

Comment:
Shouldn't it be a manual job only accessible on a tag?

Reply:
Yes, I didn't change this process, it was like this before. But I can convert it as a manual job. 
I think the initial idea, was to comment it in order to avoid any mistake

Reply:
I have check and it is already uncommented